server/src/main/scala/actors/DrtSystemInterface.scala

```diff
@@ -377,13 +379,21 @@ trait DrtSystemInterface extends UserRoleProviderLike {
         GlaFeed(liveUrl, liveToken, livePassword, liveUsername, ProdGlaFeedRequester).tickingSource
       case "PIK" =>
         CiriumFeed(config.get[String]("feeds.cirium.host"), portCode).tickingSource(30 seconds)
+      case "EDI" =>
+        val currentDate = SDate.yyyyMmDdForZone(SDate.now(), DateTimeZone.UTC)
+        val endDate = SDate.yyyyMmDdForZone(JodaSDate(new DateTime(DateTimeZone.UTC).plusDays(2)), DateTimeZone.UTC)
+        new EdiFeed(EdiClient(config.get[String]("feeds.edi.endPointUrl"), config.get[String]("feeds.edi.subscriberId"), new HttpClient)).ediFeedPollingSource(5 minutes, currentDate, endDate, LiveFeedSource)
       case _ => arrivalsNoOp
     }
 
   def forecastArrivalsSource(portCode: PortCode): Source[ArrivalsFeedResponse, Cancellable] = {
     val feed = portCode match {
       case PortCode("LHR") | PortCode("LGW") | PortCode("STN") => createArrivalFeed
       case PortCode("BHX") => BHXForecastFeedLegacy(params.maybeBhxSoapEndPointUrl.getOrElse(throw new Exception("Missing BHX feed URL")))
+      case PortCode("EDI") =>
+        val startDate = SDate.yyyyMmDdForZone(JodaSDate(new DateTime(DateTimeZone.UTC).plusDays(2)), DateTimeZone.UTC)
```

Comment:
Same thing

Reply:
We can call the forecast feed much less often as it won't be updated so frequently. So maybe something like once an hour

Reply:
👍